src/Serializers/FingerprintSerializer.php

```diff
@@ -73,22 +76,30 @@ private function serializeValuePerLanguageArray( array $array ) {
 		return $serialization;
 	}
 
+	private function serializeTerm( Term $term, array &$serialization ) {
+		$language = $term->getLanguageCode();
+		$result = array(
+			'language' => $language,
+			'value' => $term->getText(),
+		);
+		if ( $term instanceof TermFallback ) {
+			$result['language'] = $term->getActualLanguageCode();
+			$result['source'] = $term->getSourceLanguageCode();
```

Comment:
So this is serialized even if it is null? Does the unserializer require this or is it flexible enough to not fail if this is missing?